Jenkinsfile

```diff
@@ -44,7 +44,7 @@ pipeline {
     stage('Wait') {
       steps {
         echo 'Waiting for hapi-fhir-jpaserver-r4 service to reach steady state'
-        sh 'docker run -v /var/run/docker.sock:/var/run/docker.sock $AWS_ENV $GIT_ENV cicd wait hapi-fhir-jpaserver-r4 qa'
+        sh 'docker run -v /var/run/docker.sock:/var/run/docker.sock $AWS_ENV $GIT_ENV cicd wait hapi-fhir-jpaserver-r4-qa qa'
```

Comment:
This should also not be required. The `-qa` suffix is added automatically by the `wait` command.